hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsClient.java

```diff
@@ -131,7 +135,7 @@ public String getFileSystem() {
     return filesystem;
   }
 
-  protected AbfsPerfTracker getAbfsPerfTracker() {
+  public AbfsPerfTracker getAbfsPerfTracker() {
```

Comment:
Widening the method accessibility by making it public solely for testing is not a good practice. See if at least the acces can be restricted to package level and use @VisibleForTesting annotation.

Reply:
This wont be necessary as test code is refactored.